WordPress/Classes/Models/Blog+BlogAuthors.swift

```diff
@@ -17,4 +17,9 @@ extension Blog {
 
     @objc(removeAuthors:)
     @NSManaged public func removeFromAuthors(_ values: NSSet)
+
+    @objc
+    func getAuthorWith(id: NSNumber) -> BlogAuthor? {
+        return authors?.first(where: { $0.userID == id })
```

Comment:
Does the `==` here compare the user ID values or the memory address of the `NSNumber` objects?

Reply:
This should be a comparison between two `NSNumbers` now that `authors` is a Set<`BlogAuthors`> and `id` is the passed in parameter.

For better clarity I could go with `getAuthorWith(userID: NSNumber)` if you think that'd be helpful?

Reply:
Sorry, I meant to ask if
```
return authors?.first(where: { $0.userID == id })
```
is the correct way to compare `NSNumber` objects in Swift. 
Is there any difference between using `==` (as you do) and:
```
return authors?.first(where: { $0.userID.isEqual(to: id) })
```

> For better clarity I could go with getAuthorWith(userID: NSNumber). Possibly that'd be better?

I think it's fine as-is (just my personal opinion).

Reply:
Ah, I see! Sorry about that! According to [Apple's docs](https://developer.apple.com/documentation/foundation/nsnumber/1411315-isequal), `.isEqual` on a NSNumber:

> Returns a Boolean value that indicates whether the number object’s value and a given number are equal.

So I believe it only cares about the value inside and not the object itself, which would be equatable to `==`.

Reply:
I understand now, thanks!

This is what Apple's docs are showing for the `==` between two NSObjects like these, so I think it's equivalent.

```
extension NSObject : Equatable, Hashable {

    /// Returns a Boolean value indicating whether two values are
    /// equal. `NSObject` implements this by calling `lhs.isEqual(rhs)`.
    ///
    /// Subclasses of `NSObject` can customize Equatable conformance by overriding
    /// `isEqual(_:)`. If two objects are equal, they must have the same hash
    /// value, so if you override `isEqual(_:)`, make sure you also override the
    /// `hash` property.
    ///
    /// - Parameters:
    ///   - lhs: A value to compare.
    ///   - rhs: Another value to compare.
    public static func == (lhs: NSObject, rhs: NSObject) -> Bool
```

Reply:
That makes sense! Thanks for confirming and answering this question I had in the back of my mind.

Reply:
Very welcome! Thanks for bringing it up!